tests/main.cpp

```diff
@@ -1,8 +1,43 @@
+#include <string>
+
+#include "solana.hpp"
 #define DOCTEST_CONFIG_IMPLEMENT_WITH_MAIN
 #include <doctest/doctest.h>
 
 #include "MangoAccount.hpp"
 
+const std::string DEVNET = "https://api.devnet.solana.com";
+const std::string KEY_PAIR_FILE = "../tests/fixtures/solana/id.json";
+
+TEST_CASE("Simulate & Send Transaction") {
+  const solana::Keypair keyPair = solana::Keypair::fromFile(KEY_PAIR_FILE);
+  auto connection = solana::rpc::Connection(DEVNET);
+
+  // create a compiled transaction
+  auto recentBlockHash = connection.getLatestBlockhash();
+  const solana::PublicKey memoProgram =
+      solana::PublicKey::fromBase58(solana::MEMO_PROGRAM_ID);
+  const std::string memo = "Hello";
+
+  const solana::CompiledInstruction ix = {
+      1, {}, std::vector<uint8_t>(memo.begin(), memo.end())};
+
+  const solana::CompiledTransaction tx = {
+      recentBlockHash, {keyPair.publicKey, memoProgram}, {ix}, 1, 0, 1};
+
+  // call simulateTransaction
+  connection.simulateTransaction(keyPair, tx);
+  // simulate transaction
+  connection.sendTransaction(keyPair, tx);
+}
+
+TEST_CASE("Request Airdrop") {
+  const solana::Keypair keyPair = solana::Keypair::fromFile(KEY_PAIR_FILE);
+  auto connection = solana::rpc::Connection(DEVNET);
+  auto result = connection.requestAirdrop(keyPair.publicKey, 1000000000);
+  CHECK_GT(result.length(), 0);
```

Comment:
requestAirdrop should return the transaction signature of the aidrop request, not an empty response. also the wallet balance should increase

Reply:
Fixed.
P.S. had to add a 15 sec wait in the test so that the validators can process the airdrop